expressEngine/cms/product/index.js

```diff
@@ -0,0 +1,5 @@
+import ErrorMsgs from './error';
+import InfoMsgs from './info';
+import SuccessMsgs from './success';
+
+export default { ...ErrorMsgs, ...InfoMsgs, ...SuccessMsgs };
```

Comment:
unnecessary use of spread operator here.
Replace
export default { ...ErrorMsgs, ...InfoMsgs, ...SuccessMsgs }; to export default { errror, info, success };